src/mocsolver.jl

```diff
@@ -336,18 +421,23 @@ function add_q_to_φ!(sol::MoCSolution, prob::MoCProblem)
     @inbounds for i in 1:NRegions
         xs = getxs(prob, i)
         @unpack Σt = xs
-
+        if volumes[i] < 1e-12 # deal with zero volume FSRs
+            volumes[i] = 1e-12
+        end
         for g in 1:NGroups
             ig = @region_index(i, g)
             φ[ig] /= (Σt[g] * volumes[i])
-            φ[ig] += (4π * q[ig])
+            φ[ig] += (4π * q[ig]) 
+            if φ[ig] < 0.0
+                φ[ig] = 1e-12
+            end
```

Comment:
why??

Reply:
to avoid negative fluxes? please, use `const`

Reply:
Yes this is to avoid negative fluxes. I updated this to use `const`, and to use the Julia ternary conditional operator.